packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/synthesize-resources.ts

```diff
@@ -275,17 +284,23 @@ export const updateUserPoolGroups = async (context: $TSContext, resourceName: st
     ensureDirSync(userPoolGroupFolder);
     JSONUtilities.writeJson(path.join(userPoolGroupFolder, 'user-pool-group-precedence.json'), updatedUserPoolGroupList);
 
-    context.amplify.updateamplifyMetaAfterResourceUpdate(AmplifyCategories.AUTH, 'userPoolGroups', 'userPoolGroups', {
-      service: 'Cognito-UserPool-Groups',
-      providerPlugin: 'awscloudformation',
-      dependsOn: [
-        {
-          category: AmplifyCategories.AUTH,
-          resourceName,
-          attributes: ['UserPoolId', 'AppClientIDWeb', 'AppClientID', 'IdentityPoolId'],
-        },
-      ],
-    });
+    context.amplify.updateamplifyMetaAfterResourceUpdate(AmplifyCategories.AUTH, 'userPoolGroups', 'service', 'Cognito-UserPool-Groups');
+    context.amplify.updateamplifyMetaAfterResourceUpdate(AmplifyCategories.AUTH, 'userPoolGroups', 'providerPlugin', 'awscloudformation');
+
+    const parametersJSONPath = path.join(context.amplify.pathManager.getBackendDirPath(), 'auth', resourceName, 'build', 'parameters.json');
+    const authParameters = JSONUtilities.readJson<AuthParameters>(parametersJSONPath)!;
```

Comment:
nit: this can simplify to:

```suggestion
    const authParameters = stateManager.getResourceParametersJson(undefined, AmplifyCategories.AUTH, resourceName);
```

`stateManager` would need to be imported from `amplify-cli-core`

Reply:
I'll give it a try.

Reply:
Done. 
I've run `yarn run e2e --detectOpenHandles --maxWorkers=3 src/__tests__/auth_3.test.ts` locally to double check this.